src/web/lib/components/ThemeColorsEditor/index.js

```diff
@@ -1,35 +1,12 @@
 import React from 'react';
 import classnames from 'classnames';
-import { BlockPicker } from 'react-color';
+import { SketchPicker } from 'react-color';
```

Comment:
I relented :)

Reply:
I see what you mean about the caret, though 😞 - that looked nice in the other component IMO